source/_components/mqtt.markdown

```diff
@@ -23,6 +23,7 @@ To integrate MQTT into Home Assistant, add the following section to your `config
 ```yaml
 # Example configuration.yaml entry
 mqtt:
+  password: mypass
```

Comment:
This is already [covered](https://www.home-assistant.io/docs/mqtt/broker#embedded-broker). We keep the configuration sample minimal and use a password is optional.

Reply:
It doesn't work if you don't add a password though. It throws an error without the "password: mypass" entry.